### ISSUE TITLE
Add Missing Author Names

### ISSUE THREAD

**Issue by seths10** [seths10/Quotes-Generator#184]
some of the quotes do not have any author, thus they've been dubbed `anonymous`. Please add the actual author to the particular author if you know his/her name.

**Comment by MojoBoingo** [comment]
This an open issue or you looking for an assignee? Either way I'd like to work on it, researching the authors seems like a good low-code contribution. Pls assign me if that's how it goes.

**Comment by ChauhanHarshit** [comment]
hey i would like to contribute to this issue. please assign me.

**Comment by seths10** [comment]
I have assigned both of you to the task @ChauhanHarshit @MojoBoingo

**Comment by MojoBoingo** [comment]
@seths10 👌

**Comment by MojoBoingo** [comment]
hey @seths10 I'm finding a lot of duplicate quotes just among the ones with anon authors. They are the only ones I'm looking at for the moment, but it's prevalent enough I doubt it's just these. I'm noting the ones I see for deletion, can include that in the PR or not. For now I'm just leaving a comment next to duplicates in the code.

**Comment by seths10** [comment]
you can help fix it

**Comment by MojoBoingo** [comment]
deleting duplicates could be a good issue for someone :)

**Comment by seths10** [comment]
removing duplicates is already an issue

**Comment by MojoBoingo** [comment]
yeah I realized that. I haven't begun learning JS proper yet so I'll leave writing a script to someone else who needs another PR in the next 6 days. I'd be mostly doing it manually. I am leaving a comment denoting each duplicate I found though. Working on my branch as we speak, will push p soon.

**Comment by MojoBoingo** [comment]
@seths10 PR sent, found 12 quotes an author.
